core/test/element_event_operations_test.cpp

```diff
@@ -10,19 +10,19 @@
 using namespace scipp;
 using namespace scipp::core;
 
+using element::event::map_linspace;
+using element::event::map_sorted_edges;
+
```

Comment:
Additional testing does not seem to cover all element level functions such as `bin_index_linspace`

Reply:
Added to list in #1363.